### PR TITLE
solution

### PR DESCRIPTION
**Pull request by SHCHERBANV**

### CODE REVIEW

**Comment by SHCHERBANV** [comment]
@SHCHERBANV

**Review by NazarSheremet — CHANGES_REQUESTED** [review]
The pipeline should be triggered, revert the trigger back.

**Comment by SHCHERBANV** [comment]
https://github.com/SHCHERBANV/terraform_task_1_initial_setup/pull/3/commits/9f0b82aaf6e7de71486519b0490fccd0dbe992bc

**Comment by SHCHERBANV** [comment]
- name: Validate Terraform
      run: |
        if [ -d "my-terraform-project" ]; then
          cd my-terraform-project
          if ! terraform validate; then
            echo "Terraform validation failed."
            exit 1
          fi
          echo "Terraform validation succeeded."
        else
          echo "Directory my-terraform-project not found, attempting to validate .terraform.lock.hcl instead."
          if [ ! -f "my-terraform-project/.terraform.lock.hcl" ]; then
            echo "Lock file not found!"
            exit 1
          fi
          echo "Lock file found and verified."
        fi

File: .github/workflows/terraform-validate.yml [hunk 1]
```diff
@@ -20,15 +20,19 @@ jobs:
       uses: hashicorp/setup-terraform@v3
       with:
         terraform_version: 1.8.4
+    
+    - name: Terraform Init
+      run: terraform init
```
Comment (YuriiSmolii) [inline .github/workflows/terraform-validate.yml:25]:
```suggestion
      run: |
          cd my-terraform-project
          terraform init
```

**Review by YuriiSmolii — CHANGES_REQUESTED** [review]
Fix init step to pass tests

**Pull request closed without merging**